server/ServerApp.h

```diff
@@ -310,6 +314,7 @@ class ServerApp : public IApp {
     PythonServer            m_python_server;
     std::map<int, int>      m_player_empire_ids;    ///< map from player id to empire id that the player controls.
     int                     m_current_turn;         ///< current turn number
+    bool                    m_turn_expired;         ///< true when turn exceeds its timeout
```

Comment:
Does this need to be wrapped in `std::atomic` for multithreading purposes?

Reply:
No, there no need in atomics. The server works in a single main thread within `run_one` method of IO context and `HandleNextEvent` method of `ServerNetwokring`.